tests/Provider/Doctrine/Issues/Issue115Test.php

```diff
@@ -19,16 +19,15 @@
  * @internal
  *
  * @small
+ *
+ * @requires PHP >= 8.1
  */
 final class Issue115Test extends TestCase
 {
     use DefaultSchemaSetupTrait;
 
     public function testIssue115(): void
     {
-        if (\PHP_VERSION_ID < 81000) {
-            self::markTestSkipped('PHP > 8.1 is required.');
-        }
```

Comment:
you should keep this, the solution I talked previously only concerns PHPStan, these instructions are related to PHPUnit

Reply:
Impossible to use this syntax because on runtime I got Syntax error with enums
I used another way https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html

Reply:
https://github.com/DamienHarper/auditor/actions/runs/3328237934/jobs/5504011752